build.sbt

```diff
@@ -0,0 +1,63 @@
+ThisBuild / version := "1.5.2"
+ThisBuild / scalaVersion := "2.13.3"
+
+lazy val scalajs = (project in file("scalajs"))
+  .enablePlugins(ScalaJSPlugin)
+  .enablePlugins(ScalaJSWeb)
+  .settings(scalajsSettings)
+
+lazy val scalajvm = (project in file("scalajvm"))
+  .enablePlugins(PlayScala)
+  .enablePlugins(SbtWeb)
+  .settings(scalajvmSettings)
+
+lazy val scalajsSettings = Seq(
+  name := "scalajs",
+  libraryDependencies ++= Seq(
+    "org.scala-js" %%% "scalajs-dom" % "1.1.0"
+  ) ++ Seq(
+    "io.circe" %%% "circe-generic",
+    "io.circe" %%% "circe-parser"
+  ).map(_ % "0.13.0"),
+  scalacOptions ++= sharedScalacOptions,
+  scalaJSUseMainModuleInitializer := true,
+  scalaJSStage := FullOptStage
```

Comment:
I'd like this option to be `FastOptStage` in `test` and `compile` builds, but `FullOptStage` in `dist` builds. Unfortunately, I couldn't figure out how to change an option depending on the current task :( All the solutions seem to concentrate on custom tasks, and it quickly gets way over my head.

Reply:
We'll see how bad it will become when the active dev stage will roll out. For now, this is okay.